examples/Basics_Serial/Basics_Serial.ino

```diff
@@ -0,0 +1,108 @@
+/*
+ * FixMath Library - Basic usage Example with loging to the console
+ *
+ * This example code is in the public domain CONTRARY TO THE LIBRARY itself,
```

Comment:
Out of curiosity, any reason for that?

Reply:
Well, one point of examples is really to give people a functional basis to modify and build on. If the example itself was LGPL'ed, this would restrict the derived project to (L)GPL again, which sort of defeats the point of the L in LGPL in that use case.

Of course this may be an overly pedantic way of looking at things, and it may reasonably go without saying that an example is meant to be used as an example... If you feel it causes more confusion than clarification, feel free to just remove that bit.